cdc/owner_test.go

```diff
@@ -130,7 +130,7 @@ func (s *ownerSuite) TestPureDML(c *check.C) {
 
 	tables := map[uint64]schema.TableName{1: {Schema: "any"}}
 
-	changeFeedInfos := map[model.ChangeFeedID]*changeFeedInfo{
+	changeFeedInfos := map[model.ChangeFeedID]*changeFeed{
```

Comment:
```suggestion
	changeFeeds := map[model.ChangeFeedID]*changeFeed{
```
This can be shorter too